frontend/Highlighter/MultiOverlay.js

```diff
@@ -16,19 +16,35 @@ import type {DOMNode} from '../types';
 class MultiOverlay {
   win: Object;
   container: DOMNode;
+  _currentNodes: ?Array<DOMNode>;
 
   constructor(window: Object) {
     this.win = window;
     var doc = window.document;
     this.container = doc.createElement('div');
     doc.body.appendChild(this.container);
+    this._currentNodes = null;
   }
 
   highlightMany(nodes: Array<DOMNode>) {
+    this._currentNodes = nodes;
     this.container.innerHTML = '';
+    var documentTop = window.pageYOffset || document.documentElement.scrollTop;
+    var scrollAmt =
+      document.documentElement.scrollTop || document.body.scrollTop;
+
     nodes.forEach(node => {
       var div = this.win.document.createElement('div');
+      if (typeof node.getBoundingClientRect !== 'function') {
+        return;
+      }
```

Comment:
Comment nodes are included in the search, which is why the error was being thrown before this was added